src/core/utils.ts

```diff
@@ -15,54 +16,56 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { config } from '../config';
+import { config } from '../config.js';
 
-export function normalizeColor(color = '') {
-  if (Number.isInteger(color)) {
+export function normalizeColor(color: string | number = '') {
+  if (isInteger(color)) {
     return color;
   }
 
-  if (typeof color === 'string') {
```

Comment:
I did have a bug where color was not a string - I think it was boolean. But this should be ok

Reply:
TS should give you an error now if you try to do that